src/Raven.Server/Web/RequestHandler.cs

```diff
@@ -692,5 +694,22 @@ protected void RedirectToLeader()
             HttpContext.Response.Headers.Remove("Content-Type");
             HttpContext.Response.Headers.Add("Location", leaderLocation);
         }
+
+        protected PeriodicBackupConnectionType GetBackupConnectionType()
```

Comment:
can we find better place for this method? handling backup connection type in abstract request handler is IMHO not right place (to board)

Reply:
Fixed, moved to inside the controllers.